packages/ui-components/src/components/ContextMenu/ContextMenu.component.tsx

```diff
@@ -66,7 +72,7 @@ export const ContextMenu = ({
       allowedPlacements: ["top-start", "top-end", "bottom-start", "bottom-end"],
     }),
     size({
-      boundary: "viewport",
+      boundary: "viewport" as unknown as Boundary,
```

Comment:
`Boundary` type doesn't seem to accept the `string` "viewport" directly. We either need to cast, extend the type to include "viewport" etc.

Reply:
We could also use boundary: `document.documentElement`, risking changing some functionality.

Reply:
Added as improvement [here](https://github.com/cloudoperators/juno/issues/570). But since component will be deprecated, might not need to make change. Thank You, @barsukov.